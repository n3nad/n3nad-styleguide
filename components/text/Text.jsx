import React from 'react'

function Text({ children }) {
  return (
    <section>
      {children}
    </section>
  )
}

export default Text
