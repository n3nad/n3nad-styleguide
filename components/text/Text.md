Text

```jsx
<Text>
  <p>This is <b>bold text</b>, <em>this is italic</em>, and this is <a href="#">link</a>.</p>
  <ul>
    <li>Item 1</li>
    <li>Item 2</li>
    <li>Item 3</li>
  </ul>
</Text>
```

