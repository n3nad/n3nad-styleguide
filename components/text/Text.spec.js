
import React from 'react'
import { shallow } from 'enzyme'

import Text from './Text'

describe('Text component', () => {

  it("should render properly", function() {
    const wrapper = shallow(<Text><p>Some text inside</p></Text>)

    expect(wrapper).toMatchSnapshot()
  });
})
