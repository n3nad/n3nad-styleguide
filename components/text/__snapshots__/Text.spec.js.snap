// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component should render properly 1`] = `ShallowWrapper {}`;
